src/Credentials.ts

```diff
@@ -221,6 +221,22 @@ interface PersonalSignPayload {
   data: string
 }
 
+interface VC {
+  '@context': string | string[]
+  type: string | string[]
+  credentialSubject: object
+}
+
+interface VerifiableCredentialParams {
+  sub: string
+  jti: string
+  nbf: number
+  vc: VC
+  aud?: string
+  exp?: number
+  proof?: string
```

Comment:
the `proof` param should be removed. It is never used. (see https://github.com/uport-project/uport-credentials/pull/181#discussion_r303882434)

Reply:
done